script.js

```diff
@@ -1,4 +1,249 @@
+var computerTotalCards = [];
+var playerSum = 0;
+var computerSum = 0;
+​
+// Shuffle Deck
+var makeDeck = function () {
+  // Initialise an empty deck array
+  var cardDeck = [];
+  // Initialise an array of the 4 suits in our deck. We will loop over this array.
+  var suits = ["hearts", "spade", "clubs", "spade"];
+  var aceFunction = [];
+​
+  // Loop over the suits array
+  var suitIndex = 0;
+  while (suitIndex < suits.length) {
+    // Store the current suit in a variable
+    var currentSuit = suits[suitIndex];
+​
+    // Loop from 1 to 13 to create all cards for a given suit
+    // Notice rankCounter starts at 1 and not 0, and ends at 13 and not 12.
+    // This is an example of a loop without an array.
+    var rankCounter = 1;
+    var valueCounter = 1;
+    while (rankCounter <= 13) {
+      // By default, the card name is the same as rankCounter
+      var cardName = rankCounter;
+      var cardValue = valueCounter;
+      var cardValueAce1 = valueCounter;
+      var cardValueAce11 = valueCounter;
+​
+      // If rank is 1, 11, 12, or 13, set cardName to the ace or face card's name
+      if (cardName == 1) {
+        cardName = "ace";
+        cardValue = 1;
+      } else if (cardName == 11) {
+        cardName = "jack";
+        cardValue = 11;
+      } else if (cardName == 12) {
+        cardName = "queen";
+        cardValue = 12;
+      } else if (cardName == 13) {
+        cardName = "king";
+        cardValue = 13;
+      }
+​
+      // Create a new card with the current name, suit, and rank
+      var card = {
+        name: cardName,
+        suit: currentSuit,
+        rank: rankCounter,
+        value: cardValue,
+      };
+​
+      // Add the new card to the deck
+      cardDeck.push(card);
+​
+      // Increment rankCounter to iterate over the next rank
+      rankCounter += 1;
+    }
+​
+    // Increment the suit index to iterate over the next suit
+    suitIndex += 1;
+  }
+​
+  // Return the completed card deck
+  return cardDeck;
+};
+​
+var myOutputValue = `KNN CCB CAN U PUT IN THE RIGHT WORD`;
+​
+var getRandomIndex = function (max) {
+  return Math.floor(Math.random() * max);
+};
+var shuffleCards = function (cardDeck) {
+  index = 0;
+  while (index < cardDeck.length) {
+    var randomIndex = getRandomIndex(cardDeck.length);
+    var randomCard = cardDeck[randomIndex];
+    var currentCard = cardDeck[index];
+    cardDeck[index] = randomCard;
+    cardDeck[randomIndex] = currentCard;
+    index++;
+  }
+  return cardDeck;
+};
+​
+var deck = shuffleCards(makeDeck());
+​
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myOutputValue = "hello world";
+  var myOutputValue = ``;
+  var player1stCard = deck.pop();
+  var computer1stCard = deck.pop();
+  var player2ndCard = deck.pop();
+  var computer2ndCard = deck.pop();
+  var playerDraw = deck.pop();
+  var computerDraw = deck.pop();
+  var playerDraw2 = deck.pop();
+  var computerDraw2 = deck.pop();
```

Comment:
Instead of creating many variables for the cards drawn, you can create an array each for computer and players to store the cards :) Thereafter, you can create a function to create the output string to print out the card names and suits.